docs/pages/Cardinal/API-Reference/cql.mdx

```diff
@@ -0,0 +1,54 @@
+# Component Query Language
```

Comment:
In the preview link, this page appears as "Cql" in the sidebar, how can we change it to be "CQL" instead?

Reply:
You mean the file name?

Reply:
the actual item in the side bar here: https://world-engine-docs-2njn8wjng-argus-labs.vercel.app/

it's showing as `Cql`. youll have to edit the _meta.json